### PR TITLE
thrift: update urls

### PR DESCRIPTION
**Pull request by vszakats**
* use https
* use 'commitdiff_plain'
* use canonical url
* update hashes

### CODE REVIEW

File: Library/Formula/thrift.rb [hunk 1]
```diff
@@ -2,22 +2,22 @@
...
+    url "https://www.apache.org/dyn/closer.cgi?path=thrift/0.9.2/thrift-0.9.2.tar.gz"
+    sha256 "cef50d3934c41db5fa7724440cc6f10a732e7a77fe979b98c23ce45725349570"
 
     # Apply any necessary patches (none currently required)
     [
       # Example patch:
       #
       # Apply THRIFT-2201 fix from master to 0.9.1 branch (required for clang to compile with C++11 support)
-      # %w{836d95f9f00be73c6936d407977796181d1a506c 4bc8c19c51f3d9f30799251a810dd1ca63c4bf1e},
+      # %w{836d95f9f00be73c6936d407977796181d1a506c f8e14cbae1810ade4eab49d91e351459b055c81dba144c1ca3c5d6f4fe440925},
```
Comment (DomT4) [inline Library/Formula/thrift.rb]: Only one of these seems to be `sha256`? Let's make the example consistent if so.

Comment (vszakats) [inline Library/Formula/thrift.rb]: I may have misread the code, but I assumed this is a Git hash and patch checksum pair.

Comment (vszakats) [inline Library/Formula/thrift.rb]: Giving this URL, which resolves to a THRIFT-2201 patch mentioned in the comment: https://git-wip-us.apache.org/repos/asf?p=thrift.git;a=commitdiff_plain;h=836d95f9f00be73c6936d407977796181d1a506c

Comment (DomT4) [inline Library/Formula/thrift.rb]: Ah, you are right. I misread the code :wink:

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/8ce12ebbfabd55874853e1fcb895f1acf536dc28. Thanks again @vszakats!

**Pull request closed without merging**